superset-frontend/src/components/ListView/CardCollection.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { TableInstance } from 'react-table';
+import styled from '@superset-ui/style';
+
+interface Props {
+  renderCard?: (row: any) => React.ReactNode;
+  prepareRow: TableInstance['prepareRow'];
+  rows: TableInstance['rows'];
+  loading: boolean;
+}
+
+const CardContainer = styled.div`
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(459px, max-content));
```

Comment:
again wondering about the origin of the 459 value, wondering if it should be something like `calc(some% - 1px)`

And the grid-gap and padding could be `gridUnit` based.

Reply:
happy to move the spacing to use gridUnit. I pulled the style for `grid-template-column` off a stack overflow post, so I have no idea if there is a better way to do this... I just found something that's good enough

Reply:
Created [this issue](https://github.com/apache/incubator-superset/issues/10599) for these 459px widths (there are a couple in here) and self assigned it, for following up... whenever.